src/python_testing/mdns_discovery/utils/net_utils.py

```diff
@@ -0,0 +1,62 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import ipaddress
+import logging
+
+import ifaddr
+from zeroconf import InterfaceChoice
+
+logger = logging.getLogger(__name__)
+
+
+def get_ipv6_addresses():
```

Comment:
So this is the host ip addresses? If so, can you change the function name to explicitly say that? Also please add a docstring.